Taskfile

```diff
@@ -37,6 +37,7 @@ function ui() {
 
 swagger-codegen() {
   docker run --rm -it \
+    -v "$(pwd)"/php.sdk.config.json:/specs/php.sdk.config.json \
```

Comment:
I suggest creating a config dir, as examples/sdk. and potentially examples/sdk/config

Reply:
I was committed by accident =)
Now this config is located with qase sources